test/unit/support.js

```diff
@@ -59,39 +59,39 @@ testIframe(
 		userAgent = window.navigator.userAgent,
 		expectedMap = {
 			ie_11: {
-				cssSupportsSelector: false,
+				cssHas: true,
 				reliableTrDimensions: false
 			},
 			chrome: {
-				cssSupportsSelector: false,
+				cssHas: false,
 				reliableTrDimensions: true
 			},
 			safari: {
-				cssSupportsSelector: false,
+				cssHas: false,
 				reliableTrDimensions: true
 			},
 			webkit: {
-				cssSupportsSelector: true,
+				cssHas: true,
 				reliableTrDimensions: true
 			},
 			firefox_102: {
-				cssSupportsSelector: false,
+				cssHas: true,
 				reliableTrDimensions: false
 			},
 			firefox: {
-				cssSupportsSelector: true,
+				cssHas: true,
 				reliableTrDimensions: false
 			},
 			ios: {
-				cssSupportsSelector: false,
+				cssHas: false,
```

Comment:
Note to self - iOS 14 has to report `true` here; this entry needs to be split just like in #5207.